src/config.ts

```diff
@@ -322,6 +322,10 @@ export const RFQT_REQUEST_MAX_RESPONSE_MS: number = _.isEmpty(process.env.RFQT_R
     ? 600
     : assertEnvVarType('RFQT_REQUEST_MAX_RESPONSE_MS', process.env.RFQT_REQUEST_MAX_RESPONSE_MS, EnvVarType.Integer);
 
+export const ENABLE_GASLESS_RFQT: boolean = _.isEmpty(process.env.ENABLE_GASLESS_RFQT)
+    ? false
+    : assertEnvVarType('ENABLE_GASLESS_RFQT', process.env.ENABLE_GASLESS_RFQT, EnvVarType.Boolean);
```

Comment:
nit: can you use the `resolveEnvVar` instead? Trying to get people to move over to this way of loading up env variables instead 

```suggestion
export const ENABLE_GASLESS_RFQT: boolean = resolveEnvVar<boolean>(
    'ENABLE_GASLESS_RFQT',
    EnvVarType.Boolean,
    false,
);
```